### PR TITLE
Updated kg-covid-19 page per Nomi's request

### PR DESCRIPTION
**Pull request by justaddcoffee**
Updated kg-covid-19 page, per @nlharris 's request (better late than never!)

### CODE REVIEW

File: project/kg-covid-19/index.md [hunk 1]
```diff
@@ -11,9 +11,40 @@ twitter:
...
+the Human Phenotype Ontology, the Gene Ontology, and MONDO for phenotype, gene 
+annotations and diseases, respectively, in order to integrate the data in an
+ontology-aware way. 
+
+![img](kg-covid-19-contents.png)
+
+We are currently applying machine learning (ML) techniques to discover new links 
+(for example, identifying existing drugs that could be repurposed to treat COVID-19)
+and generate actionable knowledge. In addition, we are providing this knowledge graph
+to the COVID19 research community in order to accelerate COVID-19 research.
```
Comment (nlharris) [inline project/kg-covid-19/index.md:37]: COVID19 -> COVID-19

File: project/kg-covid-19/index.md [hunk 2]
```diff
@@ -11,9 +11,40 @@ twitter:
...
+
+![img](kg-covid-pipeline.png)
+
+People:
+
+Justin Reese
```
Comment (nlharris) [inline project/kg-covid-19/index.md:43]: add affiliations? e.g.
Justin Reese (LBNL)

File: project/kg-covid-19/index.md [hunk 3]
```diff
@@ -11,9 +11,40 @@ twitter:
...
 
-![img](kg-covid-pipeline.jpg)
+To address this challenge, we have produced we have produced a COVID-19 knowledge graph
+(KG) to support both DOE and international COVID-19 efforts:
+https://github.com/Knowledge-Graph-Hub/kg-covid-19/wiki
+
+This knowledge graph incorporates up-to-date data extracted from biomedical databases 
+and literature, including drug, protein-protein interactions, SARS-CoV-2 gene 
+annotations, concept and publication data from the CORD19 data set. We also ingest
+the Human Phenotype Ontology, the Gene Ontology, and MONDO for phenotype, gene
```
Comment (nlharris) [inline project/kg-covid-19/index.md:28]: it's now "Mondo" (not MONDO) (i know)
links to these ontologies would be a bonus

**Comment by justaddcoffee** [comment]
@nlharris - these are the two new figures:

kg-covid-19-contents
kg-covid-pipeline

Can you merge if this looks okay to you?

**Comment by nlharris** [comment]
looks good, thanks!

**Pull request merged**